components/Typography/typography.css

```diff
@@ -0,0 +1,86 @@
+.h1,
+h1 {
+  font-size: 3rem;
+  line-height: 4rem;
+}
+
+.h2,
+h2 {
+  font-size: 2rem;
+  line-height: 2.5rem;
+}
+
+.h3,
+h3 {
+  font-size: 1.5rem;
+  line-height: 2rem;
+}
+
+.h4,
+h4 {
+  font-size: 1rem;
+  line-height: 1.5rem;
+}
+
+.h5,
+h5 {
+  font-size: 0.5rem;
+  line-height: 1rem;
+}
+
+.font-black {
+  font-weight: 900;
+}
```

Comment:
.font-bolder would be better than .font-black

Reply:
Thanks Manikandan for suggesting these changes. I have pushed these changes.